### PR TITLE
[LinkedIn CAPI] Updates ad account ID description

### PR DESCRIPTION
**Pull request by nick-Ag**
The LinkedIn CAPI Ad Account ID field is required when creating/selecting a conversion rule to connect to the destination. This field cannot be updated once a conversion rule is created however. To communicate this to customers I've updated the description here. 

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

Testing not required: This is a non-functional change.

Screenshot 2024-09-24 at 5 05 39 PM

### CODE REVIEW

File: packages/destination-actions/src/destinations/linkedin-conversions/streamConversion/index.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ const action: ActionDefinition<Settings, Payload, undefined, HookBundle> = {
       inputFields: {
         adAccountId: {
           label: 'Ad Account',
-          description: 'The ad account to use for the conversion event.',
+          description:
+            'The ad account to use when creating the conversion event. (When updating a conversion rule, changes to this field will be ignored. Ad Account IDs cannot be updated for a conversion rule.)',
```
Comment (brennan) [inline packages/destination-actions/src/destinations/linkedin-conversions/streamConversion/index.ts:22]: Can we rephrase this to clarify that the limitation is on LinkedIn's end, e.g.

```suggestion
            'The ad account to use when creating the conversion event. (When updating a conversion rule after initially creating it, changes to this field will be ignored. LinkedIn does not allow Ad Account IDs to be updated for a conversion rule.)',
```

**Review by marinhero — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @nick-Ag PR deployed. Please confirm you are happy with the update.

**Comment by nick-Ag** [comment]
> hi @nick-Ag PR deployed. Please confirm you are happy with the update.

Looks great, Thanks Joe!
